### PR TITLE
support Ruby 3

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Review by ehelms — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -9,12 +9,18 @@ on:
...
     steps:
     - uses: actions/checkout@v2
     - name: Set up Ruby
       uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 2.7
+        ruby-version: ${{ matrix.ruby }}
         bundler-cache: true
     - name: Run Rubocop
       run: bundle exec rubocop
```
Comment (ekohl) [inline .github/workflows/test.yml:26]: Do we want to run rubocop in a matrix as well? Thoughts on just using https://github.com/theforeman/actions#rubocop & https://github.com/theforeman/actions#gem-test instead?

Comment (evgeni) [reply .github/workflows/test.yml:26]: I didn't want to disrupt the rest of the action just yet.
Matrixing Rubocop doesn't hurt. Long term we should move to shared actions, but IMHO don't need today.

**Review by ianballou — APPROVED** [review]
Looking good to me. If not done now, should we make a redmine for the rubocop matrixing?

**Comment by evgeni** [comment]
Nah, it's fine.

There is some general discussion about how we could/should test proxy plugins in https://github.com/theforeman/actions/pull/27

Depending on the outcome, I'd then see to open PRs or Redmines ;)

**Pull request merged**